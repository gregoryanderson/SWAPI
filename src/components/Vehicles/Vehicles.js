import React from 'react';
import './Vehicles.css';

function Vehicles () {
    return (
        <div>
            <p>A</p>
            <h1>L</h1>
            <h2>Y</h2>
            <h3>S</h3>
            <h4>S</h4>
            <h5>A</h5>
        </div>
    )
}

export default Vehicles;