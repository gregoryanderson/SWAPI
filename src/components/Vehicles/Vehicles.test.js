import React from 'react';
import {shallow} from 'enzyme';
import Vehicles from './Vehicles';

describe('Vehicles', () => {
    it('should match the snapshot', () => {
    const wrapper = <Vehicles />;
    expect(wrapper).toMatchSnapshot();
    })
})