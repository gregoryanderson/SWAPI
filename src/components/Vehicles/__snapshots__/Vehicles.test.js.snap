// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Vehicles should match the snapshot 1`] = `<Vehicles />`;
